src/PointwiseFunctions/Hydro/InitialData/IrrotationalBns.hpp

```diff
@@ -0,0 +1,126 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Utilities/Gsl.hpp"
+
+/*!
+ * \ingroup HydroInitialDataGroup
+ * \brief Items related to solving for irrotational bns initial data
+ * See e.g. \cite BaumgarteShapiro Ch. 15 (P. 523)
+ */
+namespace hydro::initial_data::irrotational_bns {
+/// @{
+/// \brief Compute the  shift plus a spatial vector \f$ k^i\f$ representing
+/// the local binary rotation \f$B^i = \beta^i + k^i\f$
+///
+template <typename DataType>
+void rotational_shift(
+    gsl::not_null<tnsr::I<DataType, 3>*> result,
+    const tnsr::I<DataType, 3>& shift,
+    const tnsr::I<DataType, 3>& spatial_rotational_killing_vector);
+
+template <typename DataType>
+tnsr::I<DataType, 3> rotational_shift(
+    const tnsr::I<DataType, 3>& shift,
+    const tnsr::I<DataType, 3>& spatial_rotational_killing_vector);
+/// @}
+
+/// @{
+/// \brief Compute the  stress-energy corresponding to the rotation shift
+///
+///
+/// \f[\Sigma^i_j = \frac{1}{2}\frac{B^iB_j}{\alpha^2}\f]
+///
+template <typename DataType>
+void rotational_shift_stress(gsl::not_null<tnsr::II<DataType, 3>*> result,
+                             const tnsr::I<DataType, 3>& rotational_shift,
+                             const Scalar<DataType>& lapse);
+template <typename DataType>
+tnsr::II<DataType, 3> rotational_shift_stress(
+    const tnsr::I<DataType, 3>& rotational_shift,
+    const Scalar<DataType>& lapse);
+/// @}
+
+/// @{
+/// \brief  Compute derivative  \f$D_i B^j / \alpha \f$
```

Comment:
Is this `D_i (B^j / \alpha)`? Please add parens. Also, don't we actually need `D_i (B^j / \alpha^2)` (lapse squared)?

Reply:
Good point, in the shuffling of things around this function fell out of use,  and ended up hard-coded in the initial data as a part of the fixed sources.  I'll change it to do the right thing though (what you've suggested above), because we need it for both analytic and numeric initial data.